evergreen/config.yml

```diff
@@ -867,6 +889,23 @@ buildvariants:
     distros:
     - rhel70-large
 
+- name: ubuntu-valgrind
+  display_name: "Ubuntu 2204 Valgrind"
+  run_on: ubuntu2204-large
+  expansions:
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.20.3-linux-x86_64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    fetch_missing_dependencies: On
+    enable_valgrind: On
+    cmake_build_type: RelWithDebInfo
+    curl: "/opt/mongodbtoolchain/v3/bin/curl"
+    python3: "/opt/mongodbtoolchain/v3/bin/python3"
+    ninja: "/opt/mongodbtoolchain/v4/bin/ninja"
+  tasks:
+  - name: valgrind
+    distros:
```

Comment:
nit that you don't need to specify the distro for a task if it's the same as the `run_on` for the variant.